docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/ui/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Managing Data Product using Console"
+date: "2024-01-18"
+sidebar_label: "Using the UI"
+sidebar_position: 1
+sidebar_custom_props:
+  offerings:
+    - enterprise
+    - cloud
+
+---
+
+## Creating and editing a new Data Product using Console
+
+To create a new data product, navigate to the "Data products" option in the left main menu and click the "Create data product" button. 
+
+![Create data product](images/image-1.png)
+
+A form will appear on the page. Please enter your data product information into the form.
+
+Cliking on the "Create and continue" will brig you to the definition of event specifications page:
```

Comment:
A few typos here. Should this instead say something along the lines of `Enter your data product information and click "Create and continue" to navigate to the event specification page` to avoid some repetition?

Reply:
done good suggestion